src/components/MdField/MdField.vue

```diff
@@ -34,6 +35,16 @@
     },
     props: {
       mdInline: Boolean,
+      mdNested: Boolean,
+      mdLayout: {
+        type: String,
+        default: 'normal',
+        ...MdPropValidator('md-layout', [
```

Comment:
I think this shouldn't be `md-layout` because it can be confusing with the `md-layout` classes. Maybe we can rename it to `md-type`, `md-variant` or `md-variation`

Reply:
Yes, i can implement md-dense too
Also Textarea will works.
I think we can use md-variant for layout types

I will commit tgese changes ASAP